src/Router.php

```diff
@@ -466,9 +348,34 @@ public static function process_http_request() {
 			 * @since 0.0.4
 			 */
 			self::$http_status_code = 500;
-			$response['errors']     = [ FormattedError::createFromException( $error ) ];
+			$response = [
+				'errors' => [ FormattedError::createFromException( $error ) ],
```

Comment:
@chriszarate, one thing that's been broken for a while and I just realized why is that the 2nd argument here needs to be `$debug` (true or false)

So let's update this to be:

```
'errors' => [ FormattedError::createFromException( $error, GRAPHQL_DEBUG ) ]
```